src/test/objectstore/test_bluefs.cc

```diff
@@ -131,6 +141,200 @@ TEST(BlueFS, small_appends) {
   rm_temp_bdev(fn);
 }
 
+bool stop = false;
+
+void write_data(BlueFS &fs)
+{
+    BlueFS::FileWriter *h;
+    int j=0, r=0;
+    stringstream ss;
+    string dir = "dir.";
+    ss << std::this_thread::get_id();
+    dir.append(ss.str());
+    dir.append(".");
+    dir.append(to_string(j));
+    ASSERT_EQ(0, fs.mkdir(dir));
+    while(1) {
+      string file = "file.";
+      file.append(to_string(j));
+      ASSERT_EQ(0, fs.open_for_write(dir, file, &h, false));
+      bufferlist bl;
+      char *buf = gen_buffer(4096);
+      bufferptr bp = buffer::claim_char(4096, buf);
+      bl.push_back(bp);
+      h->append(bl);
+      r = fs.fsync(h);
+      if (r < 0) {
+         std::cout << " num iterations: " << j << std::endl;
+         fs.close_writer(h);
+         break;
+      }
+      fs.close_writer(h);
+      j++;
+      if (stop)
+        break;
```

Comment:
here too

and make the loop write for N iterations or bytes (maybe specified as an arg?)